Public/Sdk/Public/Managed/Tools/Csc/csc.dsc

```diff
@@ -332,6 +333,9 @@ export interface Arguments extends Transformer.RunnerArguments{
     deterministic?: boolean;
     /** Specify a mapping for source path names output by the compiler to substitute the paths in the output PDBs*/
     pathMap?: PathMapEntry[];
+
+    /** Privilege level required by this process to execute. */
+    privilegeLevel?: "standard" | "admin";

```

Comment:
>"standard" | "admin"; [](start = 21, length = 21)

Can we not introduce a type in the Prelude, and reuse that type? Here you have the type specified in many places. If someone updates one place, then he/she should not forget updating the other places. So, please introduce a type in Prelude or in Transformers, and refer to that type.